activities/FoodChain.activity/playgame.js

```diff
@@ -255,7 +255,14 @@ enyo.kind({
 	// Show direction to frog using click on board
 	clickToMove: function(s, e) {
 		// Compute direction comparing click with frog position
-		var dx = e.clientX-this.frog.getX(), dy = e.clientY-this.frog.getY(); 
+
+	  // Calculate the position of the mouse in world coordinates from viewport coordinates
```

Comment:
Indentation is incorrect on this line.

Reply:
A tabs vs spaces issue.  I've fixed that now.